src/functions/slack-user-avatar/handler.ts

```diff
@@ -54,15 +53,11 @@ const fetchSlackUsersWithRetry = async (retries = 5): Promise<any[]> => {
 const getSlackUserAvatar: ValidatedEventAPIGatewayProxyEvent<any> = async () => {
   const { accessToken } = await Auth.getAccessToken();
   if (!accessToken) {
-    throw new Error("Timebank authentication failed");
+    throw new Error("Severa authentication failed");
   }
 
   try {
-    const timebankUsers = await TimeBankApiProvider.getTimebankUsers(accessToken);
-    if (!timebankUsers) {
-      throw new Error("No persons retrieved from Timebank");
-    }
```

Comment:
Lets comment out this code (rather than delete) and leave a TODO: comment that it needs to be updated to use SeveraUsers. It might be a useful reference for whoever implements this feature in the future.